src/main/scala/org/fortysevendeg/sparksftp/common/HiveUserData.scala

```diff
@@ -39,9 +37,10 @@ object HiveUserData {
     salariesDataFromHive.show(false)
 
     //Excerpt from the joined table
-    sparkSession.sql("select name,salary from user_salary").show(false)
+    val user_salaries = sparkSession.sql("select name,salary from user_salary")
```

Comment:
Please, use standard val names (`userSalaries`)

Reply:
Changed in 2d4f6a9082c185ecd34a7fad21ba4a79683a1bef